app/app_main.c

```diff
@@ -1051,27 +1061,31 @@ static int enable_hash(ACVP_CTX *ctx) {
     CHECK_ENABLE_CAP_RV(rv);
     rv = acvp_cap_hash_set_parm(ctx, ACVP_HASH_SHA3_224, ACVP_HASH_IN_EMPTY, 1);
     CHECK_ENABLE_CAP_RV(rv);
+    rv = acvp_cap_hash_set_domain(ctx, ACVP_HASH_SHA3_224, ACVP_HASH_MESSAGE_LEN, 0, 65536, 8);
```

Comment:
Do these new calls need a CHECK_ENABLE_CAP_RV(rv) afterwards ? Or is it just the isn't showing it ?

Reply:
You're right, thanks!